lib/eventum/class.issue.php

```diff
@@ -1802,32 +1801,6 @@ public function addAssociation($issue_id, $associated_id, $usr_id, $link_issues
     }
 
     /**
-     * Method used to remove the issue associations related to a specific issue.
-     *
-     * @param int|array $issue_id The issue ID
-     * @param int $usr_id
-     */
-    public function deleteAssociations($issue_id, $usr_id = null)
```

Comment:
Is this the wrong branch you pushed? I thought you were trying to remove Issue::removeByProjects ?